attractions/autocomplete/autocomplete.svelte

```diff
@@ -14,12 +24,45 @@
   let _class = null;
   export { _class as class };
 
+  /**
+   * An async generator of suggestions.
+   * Receives the input from the text field and is expected to yield promises that resolve to arrays of objects
+   * @type {OptionsGetter}
+   */
   export let getOptions;
```

Comment:
I suppose we can remove these props that are only passed down to `AutocompleteField` to decrease redundancy

Reply:
By "these props", what are you referring to, next to `getOptions`? I see `optionComponent` as well, and I guess that's all

Reply:
Yeah, not too much. But still